api/maven-api-core/src/main/java/org/apache/maven/api/ArtifactCoordinate.java

```diff
@@ -23,64 +23,68 @@
 import org.apache.maven.api.annotations.Nonnull;
 
 /**
- * The {@code Coordinate} object is used to point to an {@link Artifact}
- * but the version may be specified as a range instead of an exact version.
+ * Point to an {@link Artifact} but with the version specified as a range instead of an exact version.
+ * Each {@code ArtifactCoordinate} instance is basically a pointer to a file in the Maven repository,
+ * except that the version may not be defined precisely.
```

Comment:
Not sure I like the wording, since the actual pointer is the `Artifact`. `ArtifactCoordinate` is used to create those pointers by resolving the version.

I think relocations do not come into play for artifacts, but only for dependencies, at least that's what the [resolver ArtifactResolver's doc seems to indicate](https://github.com/apache/maven-resolver/blob/97dfd1c2b9deb15734d5e401807e55cd0498332a/maven-resolver-impl/src/main/java/org/eclipse/aether/impl/ArtifactResolver.java#L41-L43)

Reply:
Replaced by "Identification of an ensemble of {@link Artifact}s in a range of versions."